Marlin/ultralcd.cpp

```diff
@@ -685,30 +700,13 @@ static void lcd_move_e() {
   if (encoderPosition != 0) {
```

Comment:
You need to add forward declarations or move these before `lcd_move_menu_axis()` you moved up.
lcd_move_x()
lcd_move_y()
lcd_move_z()
lcd_move_e()